charts/accurate/README.md

```diff
@@ -0,0 +1,79 @@
+# Accurate Helm Chart
+
+## How to use Accurate Helm repository
+
+You need to add this repository to your Helm repositories:
+
+```console
+helm repo add accurate https://cybozu-go.github.io/accurate/
+helm repo update
+```
+
+## Quick start
+
+### (Optional) installing cert-manager
```

Comment:
Why is this optional?

Reply:
Fixed: https://github.com/cybozu-go/accurate/pull/11/commits/698f23bd8bab6c893bc193c1f6d25bfa120be6e8